nva-datamodel-testutils/src/main/java/no/unit/nva/model/testing/PublicationGenerator.java

```diff
@@ -74,16 +74,18 @@ public static Publication randomPublication(Class<?> publicationInstanceClass) {
         return buildRandomPublicationFromInstance(publicationInstanceClass);
     }
 
-    public static Publication randomPublicationNonDegree() {
-        var nonDegrees = PublicationInstanceBuilder.listPublicationInstanceTypes()
-                             .stream()
-                             .filter(not(PublicationGenerator::isDegree))
-                             .toList();
-        return randomPublication(randomElement(nonDegrees));
+    public static Publication ofInstanceClasses(Class<?>... publicationInstanceClasses) {
+        var publicationInstanceClass = randomElement(publicationInstanceClasses);
+        return randomPublication(publicationInstanceClass);
```

Comment:
maybe rewrite randomPublication instead of adding ofInstanceClasse

Reply:
I think it is correct as is, but perhaps `from` not `of` since we then get`PublicationGenerator.fromInstanceClasses(Class<?>...targetClasses)`. (I'd change the param name too).

Reply:
Followed @brinxmat 's approach.